lite/pom.xml

```diff
@@ -98,7 +98,6 @@
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-artifact</artifactId>
       <version>3.3.9</version>
-      <scope>test</scope>
```

Comment:
These change the scope of the ``maven-artifact`` dependency to compile, the default. Since I moved the version check from tests-only to main code, I had to move the dependency over as well.